addon/styles/_colors.scss

```diff
@@ -56,9 +72,43 @@ $frost-color-lgrey-1: #e2e2e2 !default;
 $frost-color-lgrey-2: #dbdbdb !default;
 $frost-color-lgrey-3: #f0f2f4 !default;
 $frost-color-lgrey-4: #f5f6f7 !default;
-$frost-color-lgrey-5: #f2f2f2 !default;
+$frost-color-lgrey-5: #f2f2f2 !default;  // no longer in style guide
```

Comment:
You are probably already doing this but if not, can we check with UX and see if we want to keep this one or remove it.